{"uuid":"f31b6ff5-f9fd-4778-bab1-8c6f2d6d0e90","historyId":"fe18a77735f7c5ac66bd7b3f8904e50d","fullName":"com.nhance.web.whatsApp.automation.userTestClasses.SelfService_FeatureTest.Test_SelfService_PositiveFlow","labels":[{"name":"package","value":"com.nhance.web.whatsApp.automation.userTestClasses.SelfService_FeatureTest"},{"name":"testClass","value":"com.nhance.web.whatsApp.automation.userTestClasses.SelfService_FeatureTest"},{"name":"testMethod","value":"Test_SelfService_PositiveFlow"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.nhance.web.whatsApp.automation.userTestClasses.SelfService_FeatureTest"},{"name":"host","value":"sujaya"},{"name":"thread","value":"16872@sujaya.main(1)"}],"links":[],"name":"To do the Selfservice positiveflow.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Assertion passed expected [If you wish to continue, you can choose from the above Self Service ] but found [doc2  http://tinyurl.com/yxe5wde2 If you wish to continue, you can choose from the above Self Service options.  Type 0 to go back or type # to go back to the main menu.]","trace":"java.lang.AssertionError: Assertion passed expected [If you wish to continue, you can choose from the above Self Service ] but found [doc2  http://tinyurl.com/yxe5wde2 If you wish to continue, you can choose from the above Self Service options.  Type 0 to go back or type # to go back to the main menu.]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:453)\r\n\tat com.nhance.web.whatsApp.automation.userImplementation.SelfService_Implementation.selfService_PositiveFlow(SelfService_Implementation.java:164)\r\n\tat com.nhance.web.whatsApp.automation.userTestClasses.SelfService_FeatureTest.Test_SelfService_PositiveFlow(SelfService_FeatureTest.java:34)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1600072934417,"stop":1600072989663,"parameters":[]}